node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -112,6 +112,11 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
          * to jump past the top navigation
          */
         var setUpSkipLinks = function() {
+            // Index page has a unique structure.
+            if ($('#index-page-container').length > 0) {
```

Comment:
* The target of an `#href` has to be an `id`, and we already have an `id` assigned to the content block on the index page.

* We already have an `oae-main-content` class that's applied to the `<main>` element. It would seem to be confusing to have that class on one element and an `id` with the same name associated with a different element.